IBPSA/Fluid/Actuators/Valves/TwoWayPolynomial.mo

```diff
@@ -0,0 +1,43 @@
+within IBPSA.Fluid.Actuators.Valves;
+model TwoWayPolynomial "Two way valve with polynomial characteristic"
+  extends IBPSA.Fluid.Actuators.BaseClasses.PartialTwoWayValveKv(
+    phi=l + pol_y*(1 - l));
+
+  parameter Real[:] c "Polynomial coefficients, starting with fixed offset";
+
+protected
+  Real pol_y = sum(c.*{y_actual^i for i in 0:size(c,1)-1}) "Polynomial of valve control signal";
+
+equation
+  // We provide some slack on these bounds to avoid the assert failing when
+  // small integration errors are made, e.g. in the filter.
+  assert((pol_y>=-0.2 and pol_y <=1.2),
```

Comment:
As `y_actual` can vary between 0 and 1, you better add a test in the initial equation section for these limits.
Please also add a test of the form `pol_y(y_actual[i]) < pol_y(y_actual[i+1])` for a sequence `y_actual[i] < y_actual[i+1]` to test that the curve is increasing (or better for low size(c,1), add an analytical test.